static/js/components/dashboard/ExamEnrollmentDialog.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from "react"
+import PropTypes from "prop-types"
+import Dialog from "@material-ui/core/Dialog"
+import Button from "@material-ui/core/Button"
+
+import { singleBtnDialogActions } from "../inputs/util"
+import DialogTitle from "@material-ui/core/DialogTitle"
+import DialogActions from "@material-ui/core/DialogActions"
+import DialogContent from "@material-ui/core/DialogContent"
+import type { Course } from "../../flow/programTypes"
+
+export default class ExamEnrollmentDialog extends React.Component {
+  static contextTypes = {
+    router: PropTypes.object.isRequired
+  }
+
+  props: {
+    open: boolean,
+    course: ?Course,
+    setVisibility: (v: boolean) => void
+  }
+
+  handleCalculateClick = () => {
+    const { setVisibility } = this.props
+    setVisibility(false)
+  }
+
+  render() {
+    const { setVisibility, open, course } = this.props
+    if (!course) {
+      return null
+    }
+
+    const examEnrollButton = (
+      <Button
+        key="ask-to-calculate"
+        onClick={() => {
+          course.exam_url
+            ? (window.location = course.exam_url)
+            : setVisibility(false)
+        }}
+        className="primary-button save-button calculate-button"
+      >
+        Enroll Now
+      </Button>
+    )
+    return (
+      <Dialog
+        classes={{
+          paper: "dialog exam-enrollment-dialog",
+          root:  "exam-enrollment-dialog-wrapper"
+        }}
+        open={open}
+        onClose={() => setVisibility(false)}
+      >
+        <DialogTitle className="dialog-title">Are you sure?</DialogTitle>
+        <DialogContent>
+          Are you sure you want to enroll in the exam offered{" "}
+          {course.exam_schedulable_window}? You should only click ENROLL NOW if
+          you are going to take the exam in this run. Enrolling in the course
+          but not taking the exam will count as a 0 on the proctored exam.
```

Comment:
Another change to the last line of the pop-up:
```suggestion
          but not taking the exam will count as a 0 on the proctored exam and will use up one proctored exam attempt.
```